site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,384 @@
+      </div>
+      <!-- Burger menu (visible on small screens) -->
+      <button class="navbar-toggler collapsed" type="button" data-bs-toggle="collapse" data-bs-target=".global-header-1"
+        aria-controls="global-header-1.1 global-header-1.2" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <!-- Navbar with links -->
+      <div id="global-header-1.1" class="navbar-collapse collapse me-lg-auto global-header-1">
+        <ul class="navbar-nav">
+          <li class="nav-item"><a class="nav-link active" href="#">Discover</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Shop</a></li>
+        </ul>
+      </div>
+      <!-- Navbar with action icons -->
+      <div id="global-header-1.2" class="navbar-collapse collapse d-sm-flex global-header-1">
+        <ul class="navbar-nav flex-row">
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/5.2/assets/img/boosted-sprite.svg#search"></use>
+              </svg>
+              <span class="visually-hidden">Search</span>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/5.2/assets/img/boosted-sprite.svg#buy"></use>
+              </svg>
+              <span class="visually-hidden">Basket</span>
+              <span class="position-relative align-self-start">
+                <span class="badge bg-info rounded-pill position-absolute top-0 fs-6 text-white translate-middle">
+                  1
+                  <span class="visually-hidden">shopping basket items</span>
+                </span>
+              </span>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <img src="/docs/5.2/assets/img/navbar-contact.png" width="25" height="25" role="img" alt="User"
+                loading="lazy">
+              <span class="visually-hidden">My account</span>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+</header>
+<!-- Main -->
+<main class="bd-content order-1">
+  <div class="bg-white title-bar">
+    <div class="container-xxl d-block">
+      <div class="row">
+        <div class="col-md-8 mx-auto">
+          <h1 class="display-1">Become a resseler</h1>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container-xxl pt-4">
+    <div class="row">
+      <div class="col-md-8 mx-auto">
+        <p class="ll-sm h3 mb-4">You are a professional and wish to resell Orange products to your
```

Comment:
The size seems incorrect in mobile mode. Displayed here in 16px instead of 18px.

Reply:
Similar pages with the same structure : https://boosted.orange.com/docs/5.2/guidelines/ and https://boosted.orange.com/docs/5.2/examples/

Font size for the paragraph : 24px on desktop and 16px on mobile.

I am ok to do 24px on desktop and 18px on mobile but it does not correspond to anything existing in Boosted (https://boosted.orange.com/docs/5.2/content/typography/) and it's different from what's already live.

Reply:
You need to check with the designers what should be the real value between what we have in the documentation and the page example. And we will decide what modification to do and when depending on the final rule.

Reply:
Seen wit @CyriaqueBillard, the correct font sizes are the ones in the DSM meaning that somes pages actually live are wrong and need to be updated alongside with our typography rules => subject added to our weekly specs meetings.